gemfile_lock.go

```diff
@@ -1,32 +1,21 @@
 package main
 
 import (
-	"fmt"
-	"path/filepath"
-
 	"github.com/bitrise-io/go-utils/command/gems"
-	"github.com/bitrise-io/go-utils/fileutil"
 	"github.com/bitrise-io/go-utils/log"
-	"github.com/bitrise-io/go-utils/pathutil"
 )
 
 type gemVersions struct {
 	fastlane, bundler gems.Version
 }
 
 func parseGemfileLock(searchDir string) (gemVersions, error) {
-	gemfileLockPth := filepath.Join(searchDir, "Gemfile.lock")
-	log.Printf("Checking Gemfile.lock (%s) for fastlane and bundler gem", gemfileLockPth)
-
-	if exist, err := pathutil.IsPathExists(gemfileLockPth); err != nil {
-		return gemVersions{}, fmt.Errorf("failed to check if Gemfile.lock exist at (%s), error: %s", gemfileLockPth, err)
-	} else if !exist {
-		log.Printf("Gemfile.lock does not exist")
-		return gemVersions{}, nil
-	}
-
-	content, err := fileutil.ReadStringFromFile(gemfileLockPth)
+	content, err := gems.GemFileLockContent(searchDir)
 	if err != nil {
+		if err == gems.ErrGemLockNotFound {
+			log.Printf("Gemfile.lock does not exist")
```

Comment:
it can be `gems.locked` too, pls check other logs too

Reply:
Updated.